pkgs/dxvk/installer.nix

```diff
@@ -0,0 +1,25 @@
+{ dxvk-x86
+, dxvk-x64
+, runCommandNoCC
+, writeShellScriptBin
+, pkgsCross
+}:
+let
+  setupScript = runCommandNoCC "dxvk-setup" {}
+    ''
+    cp ${dxvk-x86.src}/setup_dxvk.sh $out
+    '';
+in
+writeShellScriptBin "dxvk-installer"
+''
+  dxvk_lib32=/../../../../${dxvk-x86}/bin
+  dxvk_lib64=/../../../../${dxvk-x64}/bin
+  WINEPREFIX=''${WINEPREFIX:-"$HOME/.wine"}
+
+  . ${setupScript}
+  cp -v ${pkgsCross.mingwW64.windows.mcfgthreads}/bin/mcfgthread-12.dll "$WINEPREFIX/dosdevices/c:/windows/syswow64/"
```

Comment:
Are these the right way around? On native windows Syswow64 contains the 32 bit DLLs and system32 the 64 bit ones, despite that sounding backwards.

AFAIC mingwW64 is for the 64bit ones so it shouldn't go in syswow64.